.github/workflows/spelling.yml

```
@@ -0,0 +1,30 @@
+name: Spell checking
+on:
+  push:
+    branches:
+      - "**"
+    tags-ignore:
+      - "**"
+  pull_request_target:
+  issue_comment:
+    types: [created]
+
+jobs:
+  build:
+    name: Spell checker
+    runs-on: ubuntu-20.04
+    steps:
+    - name: checkout-merge
+      if: "contains(github.event_name, 'pull_request')"
+      uses: actions/checkout@v2.0.0
```

```suggestion
      uses: actions/checkout@v2
```
[v2.0.0](https://github.com/actions/checkout/releases/tag/v2.0.0) was released in 2019, current version is [v2.3.4](https://github.com/actions/checkout/releases/tag/v2.3.4)
Just pin to the major version.